runtime/tests/interpreter/equality_test.go

```diff
@@ -37,6 +37,22 @@ func TestInterpretEquality(t *testing.T) {
 
 		t.Parallel()
 
+		capabilityValue := interpreter.CapabilityValue{
+			Address: interpreter.NewAddressValue(common.BytesToAddress([]byte{0x1})),
+			Path: interpreter.PathValue{
+				Domain:     common.PathDomainStorage,
+				Identifier: "something",
+			},
+		}
+
+		capabilityValueDeclarartion := stdlib.StandardLibraryValue{
+			Name:       "cap",
+			Type:       &sema.CapabilityType{},
+			Value:      capabilityValue,
+			Kind:       common.DeclarationKindConstant,
+			IsConstant: true,
```

Comment:
```
			Kind:       common.DeclarationKindConstant,
			IsConstant: true,
```
Any way to get rid of this redundancy?

Reply:
Good idea! I've removed it in 3518c6d61be4f69ebf0e865d2aaad656d3e39c89 and replaced it with being inferred from the declaration kind